src/index.md

```diff
@@ -8,6 +8,93 @@ styles:
+from the network is called the service worker's **caching strategy**. There
+are a lot of strategies, and which one you use depends on the resource you're
+caching. For example, you can probably serve a logo that doesn't change often
+straight from the cache. But for a JSON file containing a user's social media
+feed, you probably want to display the cached content only while you fetch the
+latest content from the network.
+
+Caching strategy and implementation is one of the trickiest parts of
+building offline web apps. *The "core" Workbox libraries and tools*
+(`workbox-sw`, `workbox-build`, `workbox-cli`) *simplify this process.*
+
+Note: these tools and libraries are only "core" in the sense that they are
+the most popular Workbox products. Workbox also has libraries for other features,
+such as saving Google Analytics activity that occurs while a user is offline, and then
+sending that data up when the device goes back online. You can use these other
+libraries independently of `workbox-build`, etc. They don't depend on the "core" tools
+and libraries. This overview won't be covering any of these other Workbox products.
+
+`workbox-build` and `workbox-cli` generate a complete service worker for
+you at build time. By default, they generate caching strategies that work
+well for most simple apps, but you can also customize the strategies to meet
+your particular needs. That's the first main feature.
+
+The second is tracking
+whether there have been any changes to the resources that you need to save
+for offline. When a service worker first gets installed, it saves the
+current state of all the resources that you specify. But what happens
+when you update a resource after this initial installation? How do you
+get the service worker to fetch the updated version? The Workbox
+tools automate this process by tracking changes to your resources and
+ensuring that the resources get updated the next time that the app is online.
+
+If you've got a simple site and you just want it to work offline, then
+you probably only need `workbox-cli` or `workbox-build`. But if you're
+building a sophisticated site, chances are that you're going to need to
+call `workbox-sw` from your own custom service worker script, and then
+build the custom service worker script using `workbox-build`.
+
+    importScripts('/path/to/workbox-sw.js'); // not the actual filename
+    const workbox = new WorkboxSW();
+
+    // your custom service worker logic here
+
+    workbox.precache([]);
+
+Rather than generating a complete service worker for you, you use `workbox-sw`
+to handle your caching logic.
+
+`workbox.precache([])` is actually a placeholder. When you run the service worker
+through `workbox-build`, `workbox-build` actually replaces this call with an
+array of all of the resources that it's going to manage. The final service worker
+that you serve over the network looks something like this:
+
+    importScripts('/path/to/workbox-sw.js'); // not the actual filename
```

Comment:
This would benefit from us getting on board with a consistent CDN URL (`unpkg` or otherwise) and then promoting that.